File: app/src/main/java/com/zillennium/utswap/screens/setting/fundPasswordScreen/FundPasswordActivity.java
package com.zillennium.utswap.screens.setting.fundPasswordScreen;

import android.os.Bundle;
import android.widget.ImageView;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;

import com.zillennium.utswap.bases.BasePassedActivity;
import com.zillennium.utswap.databinding.ActivitySettingFundPasswordBinding;

public class FundPasswordActivity extends AppCompatActivity {

    ActivitySettingFundPasswordBinding binding;

    @Nullable
    protected void onCreate(Bundle savedInstanceState){
        super.onCreate(savedInstanceState);

        binding = ActivitySettingFundPasswordBinding.inflate(getLayoutInflater());
        setContentView(binding.getRoot());


        // Set Passed Back
        ImageView ivBack = binding.ivBack;
        BasePassedActivity.onPassedBack(FundPasswordActivity.this, ivBack);
    }

}

File: app/src/main/java/com/zillennium/utswap/screens/setting/fundPasswordScreen/FundPasswordActivity.kt
package com.zillennium.utswap.screens.setting.fundPasswordScreen

import android.content.Intent
import com.zillennium.utswap.R
import com.zillennium.utswap.UTSwapApp
import com.zillennium.utswap.bases.mvp.BaseMvpActivity
import com.zillennium.utswap.databinding.ActivitySettingCreditCardBinding
import com.zillennium.utswap.databinding.ActivitySettingFundPasswordBinding
import com.zillennium.utswap.screens.setting.addCardScreen.AddCardActivity
import java.io.IOException

class FundPasswordActivity :
    BaseMvpActivity<FundPasswordView.View, FundPasswordView.Presenter, ActivitySettingFundPasswordBinding>(),
    FundPasswordView.View {

    override var mPresenter: FundPasswordView.Presenter = FundPasswordPresenter()
    override val layoutResource: Int = R.layout.activity_setting_fund_password

    override fun initView() {
        super.initView()
        try {
            binding.apply {

                // Set Passed Back
                ivBack.setOnClickListener {
                    onBackPressed()
                }

            }
            // Code
        } catch (error: Exception) {
            // Must be safe
        }
    }
}